requirements.in

```diff
@@ -11,8 +11,10 @@ django-crispy-forms
 
 # we can probably remove tagging - taggit is used with custom models for cfp
 django-tagging==0.4.6
-django-taggit==0.22.1
-django-taggit-labels==0.6.0
+django-taggit
```

Comment:
Why are you removing the version pinning ?

Reply:
It's `django-taggit==1.2.0`